vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -39,53 +40,65 @@
 public class VimeoAccount implements Serializable {
 
     private static final long serialVersionUID = -8341071767843490585L;
-    //    private static final String TOKEN_TYPE_BEARER = "bearer";
 
+    @Nullable
     @GsonAdapterKey("access_token")
     public String accessToken;
+
+    @Nullable
     @GsonAdapterKey("token_type")
     public String tokenType;
+
+    @Nullable
     @GsonAdapterKey("scope")
     public String scope;
+
+    @Nullable
     @GsonAdapterKey("user")
     public User user;
     private String userJSON;
 
-    public VimeoAccount(){
+    public VimeoAccount() {
         //constructor for stag TypeAdapter generation
     }
 
     public VimeoAccount(@Nullable String accessToken) {
         this.accessToken = accessToken;
```

Comment:
This (pun intended) was the original code written by Alfie way back when. We need the qualifier since the parameter in the constructor and the member are the same name. It has been our practice to only add "m" to new classes (until we completely refactor the library) since this is a public library and we would need to deprecate everything properly. Since this is going in to 2.0.0 and there will be breaking changes, I could do it here. I suppose I kept with our standard here rather than changing it throughout the code base, but it's a good call out.